### PR TITLE
fix 'GoBack' button margin

### LINKED ISSUE #497: 🐞 Change `back` button look
- Environment: all
- Client:

### ✔️ Expected behavior

How it should work
https://www.figma.com/file/pxj9Hhk5AdOxXWJP0dq4gT/Library-App-Spec?node-id=4893%3A17522&t=TjgQ6wLTWnOqgldg-0

### ✖️ Actual behavior

How it works now
image

### Screenshot / Descriptions
Also check other similar buttons (`back to results`, `back to all categories`)

### PR DESCRIPTION
**Pull request by mabrur-h**
close #497 
image

### CODE REVIEW

File: packages/frontapp/src/app/components/BackButton/index.tsx [hunk 1]
```diff
@@ -7,14 +7,15 @@ import { useNavigate } from 'react-router-dom';
...
-  margin: 3rem 0 ${dimensions.xl_3} 0;
+  margin: ${dimensions.base_3} 0 ${dimensions.xl_3} 0;
   display: flex;
   align-items: center;
   cursor: pointer;
 `;
 export const GoBack = styled.a`
   font-weight: 600;
   font-size: ${dimensions.base};
+  margin-left: ${dimensions.lg};
```
Comment (Chichmarin17) [inline packages/frontapp/src/app/components/BackButton/index.tsx:18]: Can't we just use `gap` on the wrapper element instead of margin?

Comment (mabrur-h) [reply packages/frontapp/src/app/components/BackButton/index.tsx:18]: Yeah, I think we can open a space by giving gap to the wrapper. Should I change it or it is ok?

Comment (sabrina34352) [reply packages/frontapp/src/app/components/BackButton/index.tsx:18]: @mabrur-h yes change it, if it will work the same way as margin-left does

**Review by sabrina34352 — APPROVED** [review]

**Review by Chichmarin17 — APPROVED** [review]

**Comment by ngame2011** [comment]
`packages/frontapp/src/app/components/BackButton/index.tsx`

@mabrur-h , @Chichmarin17, @sabrina34352 pay attention to markup. There is a potential issue:
Semantically incorrect markup - the clickable element is a regular `div` element. To understand the problem, try covering this component with a unit test. How will you find the element to click on? In such cases, you need to add the `role="button"`  to the `div` element, but then it makes no sense to add the `link` element inside the markup.

I think that we need to create a separate task for refactoring this component

**Pull request merged**